Tests/scripts/get_previous_master_sha.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Default value
+FAIL_IF_NOT_FOUND=false
+
+# Parse command-line arguments
+while [[ $# -gt 0 ]]; do
+    key="$1"
+
+    case $key in
+        -f|--fail)
+            FAIL_IF_NOT_FOUND=true
+            shift
+            ;;
+        *)
+            echo "Unknown option: $1"
+            exit 1
+            ;;
+    esac
+done
+
+if [[ $CI_COMMIT_BRANCH = "master" ]]; then
```

Comment:
Maybe you mean this? 
```suggestion
if [[ $CI_COMMIT_BRANCH == "master" ]]; then
```